pylintrc

```diff
@@ -19,7 +19,7 @@ docstring-min-length=3
 ignored-modules=numpy,cv2
 
 [MESSAGE CONTROL]
-disable=no-member, no-value-for-parameter, no-self-use, too-many-branches, unsubscriptable-object, redefined-variable-type, too-many-star-expressions, duplicate-code, not-context-manager, too-many-lines, global-statement, locally-disabled, wrong-import-position, invalid-sequence-index, redundant-keyword-arg, bad-super-call, no-self-argument, redefined-builtin, arguments-differ, len-as-condition, keyword-arg-before-vararg, assignment-from-none, useless-return, useless-import-alias, unnecessary-pass, cyclic-import, assignment-from-no-return, comparison-with-callable, unnecessary-lambda, import-outside-toplevel, import-error
+disable=no-member, no-value-for-parameter, no-self-use, too-many-branches, unsubscriptable-object, redefined-variable-type, too-many-star-expressions, duplicate-code, not-context-manager, too-many-lines, global-statement, locally-disabled, wrong-import-position, invalid-sequence-index, redundant-keyword-arg, bad-super-call, no-self-argument, redefined-builtin, arguments-differ, len-as-condition, keyword-arg-before-vararg, assignment-from-none, useless-return, useless-import-alias, unnecessary-pass, cyclic-import, assignment-from-no-return, comparison-with-callable, unnecessary-lambda, import-outside-toplevel, import-error, abstract-method, missing-function-docstring
```

Comment:
Ignoring missing docstrings would result in no docstrings whatsoever.

Reply:
Perhaps `no-docstring-rgx` might be used to exclude certain methods like `forward`, etc